app/javascript/src/Navigation/components/header/DocumentationMenu.jsx

```diff
@@ -1,11 +1,21 @@
-import React from 'react'
+// @flow
+
+import * as React from 'react'
 import { DocumentationMenuItem } from 'Navigation/components/header'
+import { DocsProps } from 'Navigation/components/header/types'
+
+type ItemType = {
```

Comment:
This is a bit confusing and also redundant, since it's the same type definition as in DocumentationMenuItem. I think either we import this type from that class or we remove `renderMenuItems`. (My preference for the latter)